jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/expression/temporal/CurrentDateCommand.java

```diff
@@ -1,4 +1,4 @@
-package org.jbehavesupport.core.internal.expression;
+package org.jbehavesupport.core.internal.expression.temporal;
```

Comment:
well, moving old classes into the new package is potentially breaking change

Reply:
oops, sorry I overlooked the internal package, then that change is absolutely fine